validate/setup-guides/aws/createEC2.md

```diff
@@ -63,110 +75,52 @@ The term *new* is used simply because (at the time of this documentation creatio
 
 ![](/img/validator_nodes/node-aws-ec2-4.png)
 
-#### LAUNCH INSTANCE CREATION WIZARD
+### Launch Creation Wizard
 
 Choose **Launch instances** from the `top right` of the dashboard. This will tell AWS to guide us through the launch of an instance via their `launch wizard`.
 
 ![](/img/validator_nodes/node-aws-ec2-5.png)
 
-    - We will choose `Ubuntu 20.04 LTS` from the list.
-    - We will choose `ARM` from the 64-bit option type.
-    - We will click `Select`.
-
-:::note
-You can use any Debian distribution you would like, it does not need to be Ubuntu. We will be using Ubuntu throughout our tutorial(s), so if you choose a different distribution, you may need to change some commands to match your chosen distribution.
-:::
+### Name your instance
 
+![](/img/validator_nodes/node-aws-ec2-name-tag.png)
 :::tip
 It is **highly** recommended (but not required) to use a `Debian` distribution.
 :::
 
-![](/img/validator_nodes/node-aws-ec2-6.png)
-
-#### INSTANCE TYPE
-
-Click on **All instance families** to drop down the list
-
-![](/img/validator_nodes/node-aws-ec2-7.png)
-
-We will choose `a1`. Constellation Network wants us to use `a1.xlarge`
-
-Click on **`a1.xlarge`**.
-
-![](/img/validator_nodes/node-aws-ec2-8.png)
-
-Click on Next: `Configure instance Details`.
-
-
-![](/img/validator_nodes/node-aws-ec2-9.png)
-
-#### EC2 Details
-
-Need to supply AWS some details
-
-Switch our Auto-assign Public IP to Disabled. We are going to create an EIP later in this process.
+### Pick EC2 Image 
 
-![](/img/validator_nodes/node-aws-ec2-10.png)
-
-#### STORAGE
-
-Click on Next: `Add Storage`.
-
-![](/img/validator_nodes/node-aws-ec2-11.png)
-
-Change the storage size to **`160`**.
-
-![](/img/validator_nodes/node-aws-ec2-12.png)
-
-
-The next 2 steps are **Add Tags** and **Configure Security Groups**; however, *we will skip these for now*, and come back to configure them later on in our tutorial steps.
-
-Let's get our instance up and running. Select **Review and Launch**!
-
-![](/img/validator_nodes/node-aws-ec2-13.png)
-
-We will continue configuration while it is launching
-
-:::danger Possible Warning
-
-We may see these warnings, that our Node is vulnerable. We can safely ignore these because our Node will not have an IP address and can not access the Internet, with its current configuration.
-
-We will only be able to work with this EC2 instance from our dashboard only. Do not worry though, when we are done, our communications will be fixed; as well as, our security setup.
-
-![](/img/validator_nodes/node-aws-ec2-14.png)
-
-This is OK, we will fix later! 
+:::note
```

Comment:
I think these notes just add clutter to the tutorial. Why don't we just show what we show in the tutorial and not worry about customizing the linux distro? It would make this easier to follow.